post.hbs

```diff
@@ -46,27 +46,81 @@ into the {body} of the default.hbs template --}}
             {{/if}}
 
             <footer class="post-full-footer">
-                {{!-- Everything inside the #author tags pulls data from the author --}}
-                {{#author}}
 
-                <section class="author-card">
-                    {{#if profile_image}}
-                        <img class="author-profile-image" src="{{profile_image}}" alt="{{name}}" />
-                    {{/if}}
-                    <section class="author-card-content">
-                        <h4 class="author-card-name"><a href="{{url}}">{{name}}</a></h4>
-                        {{#if bio}}
-                            <p>{{bio}}</p>
+                {{!-- If there are multiple authors for the post, display all of their avatars in a stack --}}
+                {{#has author="count:>1"}}
+
+                    <section class="authors">
+
+                        <div class="names">
+                            <p>This post was a collaboration between</p>
+                            <p>{{authors}}</p>
+                        </div>
+
+                        <ul class="author-list">
+                        {{#foreach authors}}
+                            <li class="author-list-item">
+
+                                <div class="author-card">
+                                    <div class="basic-info">
```

Comment:
.author-card-content ?

Reply:
I probably wouldn't change this because it has common styles with the single author footer styles. another option is to change `author-list-item` > `author-card` and `author-card` > `author-card-content` (this requires a little bit of refactor though)